exauq/__main__.py

```diff
@@ -46,6 +56,14 @@ def main():
             action="store_true",
             help="open a browser at the EXAUQ documentation and exit",
         )
+        parser.add_argument(
+            "-v",
+            "--version", 
+            action="version",
+            version=f"exauq v{get_version()}",
```

Comment:
```suggestion
            version=f"exauq {get_version()}",
```
I suggest removing the 'v' so inline with other packages e.g python --version = `Python 3.12.6`